loops-and-arrays.js

```diff
@@ -23,6 +30,17 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+for (let i = 0; i < numbers.length; i++) {
+    if (i % 4 === 0) {
+        numbers[i] = numbers[i] * 1;
+    } if (i % 4 === 0) {
+        numbers[i] = numbers[i] * 2;
+    } if (i % 4 !== 0) {
+        numbers[i] = numbers[i] * 3;
+    }
```

Comment:
Je zou dit iets makkelijker kunnen schrijven. Sowieso is een getal * 1 altijd hetzelfde (`4 * 1 = 4`)

Daarnaast kijk je niet naar het daadwerkelijke getal, maar naar de index. Beiden heb ik verwerkt in de volgende suggestie:
```suggestion
    if (numbers[i] % 2 === 0) {
        numbers[i] = numbers[i] * 2;
    } else {
        numbers[i] = numbers[i] * 3;
    }
```

Reply:
Dank je wel Zwen voor het nakijken.